From 843e405184f9869ad4782992a9a174ea38483ac4 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 8 Jan 2025 10:09:15 +0100
Subject: [PATCH 1425/1695] dts: st: add access-controllers property in
 lp-timer 2-5 nodes on stm32mp211

Add access-controllers property in LPTIM2..5 nodes.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 08c3776ffb99aeafa90470e7e79582ae57aa7115 dts: st: add access-controllers property in lp-timer 2-5 nodes on stm32mp211]
Change-Id: I3aa620b0c8727ead48931b864eea01300a188be5
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/427488
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index ae4691242..b861419b9 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -2034,6 +2034,7 @@
 				clock-names = "mux";
 				#address-cells = <1>;
 				#size-cells = <0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_LPTIM2_ID>;
 				status = "disabled";
 
 				counter {
@@ -2274,6 +2275,7 @@
 				clock-names = "mux";
 				#address-cells = <1>;
 				#size-cells = <0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_LPTIM3_ID>;
 				status = "disabled";
 
 				counter {
@@ -2307,6 +2309,7 @@
 				clock-names = "mux";
 				#address-cells = <1>;
 				#size-cells = <0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_LPTIM4_ID>;
 				status = "disabled";
 
 				counter {
@@ -2340,6 +2343,7 @@
 				clock-names = "mux";
 				#address-cells = <1>;
 				#size-cells = <0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_LPTIM5_ID>;
 				status = "disabled";
 
 				counter {
-- 
2.47.2

